netbox/core/tables/plugins.py

```diff
@@ -37,3 +41,68 @@ class Meta(BaseTable.Meta):
         default_columns = (
             'name', 'version', 'package', 'description',
         )
+
+
+class PluginVersionTable(BaseTable):
+    version = tables.Column(
+        verbose_name=_('Version')
+    )
+    last_updated = tables.Column(
+        accessor=tables.A('date'),
+        verbose_name=_('Last Updated')
+    )
+    min_version = tables.Column(
+        accessor=tables.A('netbox_min_version'),
+        verbose_name=_('Minimum NetBox Version')
+    )
+    max_version = tables.Column(
+        accessor=tables.A('netbox_max_version'),
+        verbose_name=_('Maximum NetBox Version')
+    )
+
+    class Meta(BaseTable.Meta):
+        empty_text = _('No plugin data found')
+        fields = (
+            'version', 'last_updated', 'min_version', 'max_version',
+        )
+        default_columns = (
+            'version', 'last_updated', 'min_version', 'max_version',
+        )
+        orderable = False
+
+    def render_last_updated(self, value, record):
+        return naturalday(datetime.fromisoformat(value))
+
+
+class CertifiedPluginTable(BaseTable):
```

Comment:
This should be renamed, as not all catalog plugins have been certified.
```suggestion
class CatalogPluginTable(BaseTable):
```